admin/invoice.php

```diff
@@ -459,7 +460,12 @@ function couponpopupWindow(url) { /* just a stub for coupon output that might fi
           }
       ?>
     </div>
-
+<?php
+} else {
+    // no information for invoice
+    echo ERROR_ORDER_DOES_NOT_EXIST . $oID;
+}
+?>
```

Comment:
Put the message in messagestack and redirect to orders, just like in admin/orders.php line 70-72.  No sense keeping them on this page.